<!DOCTYPE html>
<html lang="en">
<head>
     <meta charset="utf-8" />
     <title>Rule Documentation: variant_annotator2</title>
     <link href="../assets/bootstrap.css" rel="stylesheet">
<style type="text/css">
     pre {
          display: block;
          font-family: monospace;
          white-space: pre;
          background-color:white;
          border: none;
          margin: 1em 0; }
     body {
          padding-left: 32px;
          padding-right: 32px;
          min-width: 800px;
</style>
</head> 
<body>
     <!-- Code Parameter -->
     <div style="border-radius: 10px; background-color:Gainsboro; padding: 10px;">
     <p><b>variant_annotator2.rl</b> - Description
     <br>Version: x.x</p>
     <div style="border-radius: 10px; background-color:white; ; padding: 10px;">
     <font color="red">
     <font color="gray"># Comment</font>
     <pre><font color="red">
rule variant_annotator:
     input: vcf="combined.gvcf"
     output: "combined.annot.gvcf"
     params: gatk=config['bin']['GATK'],genome=config['references']['GENOME'],snpsites=config['references']['SNPSITES']
     shell: "{params.gatk} -T VariantAnnotator \
    -R {params.genome} \
    -o {output} \
    -A Coverage \
    -A ChromosomeCounts \
    -A HaplotypeScore \
    -A MappingQualityRankSumTest \
    -A MappingQualityZero \
    -A RMSMappingQuality \
    -A SpanningDeletions \
    -A TandemRepeatAnnotator \
    -A FisherStrand \
    -A QualByDepth \
    -A BaseQualityRankSumTest \
    -A ReadPosRankSumTest \
    -A StrandOddsRatio \
    --variant {input.vcf} \
    -L {input.vcf} \
    --dbsnp {params.snpsites};"

</font></pre>
</div>
</div>
</body>
</html>
