<!DOCTYPE html>
<html lang="en">
<head>
     <meta charset="utf-8" />
     <title>Rule Documentation: apply_variant_recal</title>
     <link href="../assets/bootstrap.css" rel="stylesheet">
<style type="text/css">
     pre {
          display: block;
          font-family: monospace;
          white-space: pre;
          background-color:white;
          border: none;
          margin: 1em 0; }
     body {
          padding-left: 32px;
          padding-right: 32px;
          min-width: 800px;
</style>
</head>
<body>
     <!-- Code Parameter -->
     <div style="border-radius: 10px; background-color:Gainsboro; padding: 10px;">
     <p><b>apply_variant_recal.rl</b> - Description
     <br>Version: x.x</p>
     <div style="border-radius: 10px; background-color:white; ; padding: 10px;">
     <font color="red">
     <font color="gray"># Comment</font>
     <pre><font color="red">
rule apply_variant_recal:
	input: vcf="combined.gvcf","all.snp.recal", "all.snp.tranches","all.indel.recal", "all.indel.tranches"
        output: "all.vqsr.snp.vcf", "all.vqsr.indel.vcf"
        params: GATK=config['bin']['GATK'], REF=config['references']['GENOME']
        run:
          for mode in ["snp","indel"]:
              rfile="all."+mode+".recal"
              tfile="all."+mode+".tranches"
              ofile="all.vqsr."+mode+".vcf"
	      shell ("{params.GATK} -T ApplyRecalibration -R {params.REF} -input {input.vcf} --ts_filter_level 99.0 -tranchesFile "+tfile+" -recalFile "+rfile+" -mode "+mode+" -o "+ofile)

</font></pre>
</div>
</div>
</body>
</html>
