<!DOCTYPE html>
<html lang="en">
<head>
     <meta charset="utf-8" />
     <title>Rule Documentation: callvariant</title>
     <link href="../assets/bootstrap.css" rel="stylesheet">
<style type="text/css">
     pre {
          display: block;
          font-family: monospace;
          white-space: pre;
          background-color:white;
          border: none;
          margin: 1em 0; }
     body {
          padding-left: 32px;
          padding-right: 32px;
          min-width: 800px;
</style>
</head>
<body>
     <!-- Code Parameter -->
     <div style="border-radius: 10px; background-color:Gainsboro; padding: 10px;">
     <p><b>callvariant.rl</b> - Description
     <br>Version: x.x</p>
     <div style="border-radius: 10px; background-color:white; ; padding: 10px;">
     <font color="red">
     <font color="gray"># Comment</font> 
     <pre><font color="red">
rule callvariant:
     input: lambda wildcards: config['project']['units'][wildcards.x]+".recal.bam"
     output:"{x}.g.vcf"
     params: gatk=config['bin']['GATK'],genome=config['references']['GENOME'],snpsites=config['references']['SNPSITES']
     shell: "{params.gatk} -T HaplotypeCaller -R {params.genome} -I {input} --read_filter BadCigar --emitRefConfidence GVCF --annotation Coverage -A FisherStrand -A HaplotypeScore -A MappingQualityRankSumTest -A QualByDepth -A RMSMappingQuality -A ReadPosRankSumTest --variant_index_type LINEAR --dbsnp {params.snpsites} --variant_index_parameter 128000 -o {output[0]}"




#--annotation Coverage -A FisherStrand -A HaplotypeScore -A MappingQualityRankSumTest -A QualByDepth -A RMSMappingQuality -A ReadPosRankSumTest

</font></pre>
</div>
</div>
</body>
</html>
