<!-- ######################################### SECTION ######################################### -->
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <title>Samtools Suite</title>
        <link href="../assets/bootstrap.css" rel="stylesheet">
            <style type="text/css">
                body {

                    padding-top: 32px;
                    padding-bottom: 32px;
                    padding-left: 32px;
                    padding-right: 32px;
                    min-width: 1000px;
                    max-width: 1200px;
                }

            iframe {
                scrolling="no";
                width: 1000px;
                border: none;
                seamless;
            }
            	</style>

    </head>
    <body>
        <hr>
        <h3>Samtools</h3>
        <p><b>Samtools</b></p>
        <blockquote cite="http://www.htslib.org">
            <p><i>Samtools is a suite of programs for interacting with high-throughput sequencing data. It consists of three separate repositories:</i></p>
<ul>
	<li>Samtools Reading/writing/editing/indexing/viewing SAM/BAM/CRAM format</li>
	<li>BCFtools Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence variants</li>
	<li>HTSlib A C library for reading/writing high-throughput sequencing data</li>
</ul>

<br>Samtools and BCFtools both use HTSlib internally, but these source packages contain their own copies of htslib so they can be built independently
            <p>Genome Research Limited</p>
        </blockquote>

<h5>Citations:</h5>
<ul>
	<li>Li H., Handsaker B., Wysoker A., Fennell T., Ruan J., Homer N., Marth G., Abecasis G., Durbin R. and 1000 Genome Project Data Processing Subgroup (2009) The Sequence alignment/map (SAM) format and SAMtools. Bioinformatics, 25, 2078-9. [PMID: 19505943]</li>
	<li>Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics. 2011 Nov 1;27(21):2987-93. Epub 2011 Sep 8. [PMID: 21903627] Danecek P., Schiffels S., Durbin R. Multiallelic calling model in bcftools (-m)</li>
	<li>Li H. Improving SNP discovery by base alignment quality. Bioinformatics. 2011 Apr 15;27(8):1157-8. doi: 10.1093/bioinformatics/btr076. Epub 2011 Feb 13. [PMID: 21320865]</li>
	<li>Durbin R. Segregation based metric for variant call QC</li>
	<li>Li H, Mathematical Notes on SAMtools Algorithms</li>
</ul>

        <p>
        <a class="btn" href="http://www.htslib.org" target="_blank">Visit Samtools Website »</a>
        <a class="btn" href="http://www.htslib.org/doc/#manual-pages" target="_blank">Samtools Documentation »</a>
        <a class="btn" href="http://www.htslib.org/download/" target="_blank">Samtools Download »</a>
        
        
        
        </p>
    </body>
</html>
